src/SamplesApp/Benchmarks.Shared/Suite/Windows_UI/ColorBench/XamlControlsResourcesCreationBenchmark.cs

```diff
@@ -42,3 +43,4 @@ public void Create_Color_Equals()
 		}
 	}
 }
+#pragma warning restore CS0219
```

Comment:
It's not really required to restore this at the end: it won't apply to other files like compilers in the old times ;-)